src/parser/CarsFileReader.java

```diff
@@ -0,0 +1,26 @@
+package parser;
+
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class CarsFileReader {
+    private List<String> cars = new ArrayList<>();
+    private static final Logger LOGGER = Logger.getLogger(CarsFileReader.class.getName());
+
+    public List<String> parseCarList(final String filePath) {
+        try {
+            cars = Files.readAllLines(Paths.get(filePath));
```

Comment:
Return immediately. You dont't need to store cars in file reader class. It's only reader. Not storage.